src/app/core/services/external-cluster.ts

```diff
@@ -216,11 +217,27 @@ export class ExternalClusterService {
     return this._http.get<string[]>(url, {headers: this._getAKSHeaders(location)}).pipe(catchError(() => of<[]>()));
   }
 
+  getAKSKubernetesVersions() {
```

Comment:
There should be return type here. Same comment for `getEKSKubernetesVersions` method.

Reply:
@ahmadhamzh return type of `getEKSKubernetesVersions` is still missing.

Reply:
please resolve this